log4j-api-test/src/main/java/org/apache/logging/log4j/test/junit/Resources.java

```diff
@@ -20,11 +20,36 @@
 
 /**
  * Constants to use the {@link ResourceLock} annotation.
- *
  */
-public class Resources {
+public final class Resources {
```

Comment:
This class is used in `@ResourceLock` JUnit 5 annotations.

Thanks to the `DI`,  most tests in Log4j Core 3.x will not require to access service classes through static methods (yes, even for `ThreadContext`), but we want to mark those that need to be rewritten.